playground/OpenAIEndToEnd/OpenAIEndToEnd.WebStory/Components/Pages/Home.razor

```diff
@@ -1,39 +1,36 @@
 ﻿@page "/"
-@using OpenAI
-@using OpenAI.Chat
-@inject OpenAIClient aiClient
+@using Microsoft.Extensions.AI;
+@* @inject IChatClient chatClient *@
 @inject ILogger<Home> logger
 @inject IConfiguration configuration
 
     <div class="storybox" style="margin: 25%">
-        @foreach (var message in chatMessages.OfType<AssistantChatMessage>())
+        @foreach (var message in chatMessages.Where(x => x.Role == ChatRole.Assistant))
         {
-            <p style="font-size: 3em;">@message.Content[0].Text</p>
+            <p style="font-size: 3em;">@message.Text</p>
         }
 
         <button @onclick="GenerateNextParagraph" autofocus>Generate</button>
     </div>
 
 @code {
-    private ChatClient? chatClient = null;
-    private List<ChatMessage> chatMessages = new List<ChatMessage>();
+    [Inject(Key = "modelA1")]
+    public required IChatClient chatClient { get; set; }
```

Comment:
I think in this page we should only use `ChatClient` and not `IChatClient`, because it's meant to demonstrate usage of the concrete OpenAI client types directly rather than going through MEAI abstractions. It's the other page, `UseIChatClient`, that's meant to show usage of MEAI abstractions.

Reply:
Reverted the changes. Now testing with the `/useichatclient` page you added